sky/data/storage_utils.py

```diff
@@ -63,6 +66,43 @@ def format_storage_table(storages: List[Dict[str, Any]],
         return 'No existing storage.'
 
 
+def get_excluded_files_from_skyignore(src_dir_path: str) -> List[str]:
```

Comment:
nit: performance for this function can be a concern. We should test with a folder with a large amount of files and sub-folders (fine to do it after this PR).